conmon-rs/common/proto/conmon.capnp

```diff
@@ -26,9 +26,15 @@ interface Conmon {
     }
 
     struct LogDriver {
+        # The type of the log driver.
         type @0 :Type;
+
+        # The filesystem path of the log driver, if required.
         path @1 :Text;
 
+        # The maximum log size in bytes, 0 means unlimited.
+        maxSize @2 :UInt32;
```

Comment:
Let's future proof this and make the maxSize a :UInt64.

Reply:
Yep, done :+1: